java-example/README.md

```diff
@@ -39,7 +39,7 @@ Follow these steps to install the Wavefront proxy using Docker. See [Install a P
     docker ps
     ```
 
-    If `docker ps` does not list the Wavefront proxy, it means that the Wavefront proxy stopped running. If this happens, use `docker logs <container ID>` to view the logs and find the issue. The docker command you ran in step 3 prints out the container ID.
+    If `docker ps` does not list the Wavefront proxy, it means that the Wavefront proxy stopped running. If this happens, use `docker logs <container ID>` to view the logs and find the issue. The `docker` command you ran in step 3 prints out the container ID.
```

Comment:
Just curious. Is there a way in markup to make `step 3` be dynamic so that if you insert new steps before it, it updates? We wouldn't want the step number here to be wrong if someone else adds a new step later.